.github/CODEOWNERS

```diff
@@ -22,13 +26,23 @@
 /libraries/Adapters/**              @garypretty
 
 # Adaptive Expressions
-/libraries/AdaptiveExpressions/**   @tomlm @carlosscastro @boydc2014 @chrimc62
+/libraries/AdaptiveExpressions/**   @microsoft/bf-adaptive @microsoft/bb-dotnet @microsoft/bf-admin
 
 # Platform Integration Libaries
-/libraries/integration/Microsoft.Bot.Builder.Integration.ApplicationInsights.Core/**           @garypretty
-/libraries/integration/Microsoft.Bot.Builder.Integration.ApplicationInsights.WebApi/**         @garypretty
-/libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/**                        @johnataylor
-/libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.WebApi/**                      @johnataylor
+/libraries/integration/Microsoft.Bot.Builder.Integration.ApplicationInsights.Core/**           @garypretty @microsoft/bb-dotnet @microsoft/bf-admin
+/libraries/integration/Microsoft.Bot.Builder.Integration.ApplicationInsights.WebApi/**         @garypretty @microsoft/bb-dotnet @microsoft/bf-admin
+/libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/**                        @johnataylor @microsoft/bb-dotnet @microsoft/bf-admin
+/libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.WebApi/**                      @johnataylor @microsoft/bb-dotnet @microsoft/bf-admin
+
+# Adaptive Dialogs
+/libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/**                                @microsoft/bf-adaptive @microsoft/bb-dotnet @microsoft/bf-admin
+/libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams/**                          @microsoft/bf-adaptive @microsoft/bb-dotnet @microsoft/bf-admin
+/libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/**                        @microsoft/bf-adaptive @microsoft/bb-dotnet @microsoft/bf-admin
+#   Adaptive Dialogs' tests
+/tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Profiling/**                          @microsoft/bf-adaptive @microsoft/bb-dotnet @microsoft/bf-admin
+/tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams.Tests/**                        @microsoft/bf-adaptive @microsoft/bb-dotnet @microsoft/bf-admin
+/tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/**                    @microsoft/bf-adaptive @microsoft/bb-dotnet @microsoft/bf-admin
+/tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Tests/**                              @microsoft/bf-adaptive @microsoft/bb-dotnet @microsoft/bf-admin
 
 # BotBuilder Libaries
```

Comment:
Should the owners of the specific libraries be repeated for their corresponding test folders?
Just a thought: how about anyone can accept changes on tests so we don't block on that an anyone can contribute to code quality and test coverage?

Reply:
The reptition of the bb-dotnet and bf-admin don't seem to be needed. As it sits, I think that means we would need multiple signoffs. I think the adaptive team alone would be (mostly) sufficent.

Reply:
@gabog  I think we will need to do that, as chaning code in X also means changing code in X.Tests. The ownership should be the same.

Reply:
@gabog, shouldn't the tests have the same or more importance than the code? Accepting tests for incorrect behavior wouldn't help us in the long run.

The code owners is so that the correct people accept PRs, it doesn't block anyone from contributing to code quality and test coverage...

Regarding the concern around multiple reviews, this isn't applicable via code owners, but instead via the branch settings for the repository. In https://github.com/microsoft/botbuilder-dotnet/pull/3890, 8 code owners were requested for review (_55 files changed_), but only 2 reviews were necessary. This is because one of the approvals didn't satisfy the remaining code owners requirements.